### DISCUSSION TITLE
Can i call a function which i created in unity from my React js app.

### DISCUSSION THREAD

**Discussion by harmanpreet7916** [jeffreylanters/react-unity-webgl#540] [Question and Answer] [answered]
i want to call the sendToUnity function while one parameter is it possible to do from react web app

unity code in bridge file : -   

 SendtoJS(messageToSend); // Send the message to JavaScript if in WebGL build and not in Unity Editor
#endif
    }

    public void SendToUnity(string partName)
    {
        // Implement logic to handle data sent from JavaScript to Unity
        outlineSelection.ShowOutlineForTime(partName);
    }
}

i want to call this sendToUnity function with one parameter is it possible to do. from unity app

react code

![image](https://github.com/jeffreylanters/react-unity-webgl/assets/77985528/eef7b3ed-40f8-4366-814c-059be7dd1576)

i don't know what else to add in the question to give more context. please help me with this am i doing it right or there is any other way possible

what i need 
i am creating a react web application in that i am adding unity game. in that game i want to highlight the parts of the objects. but the buttons to highlight should be in react website

**Comment by jeffreylanters** [comment, marked as answer]
Hello, indeed, it is possible to achieve this by utilizing the `SendMessage` function provided by the Unity Context hook. Additional examples and detailed explanations are available in the documentation.

https://react-unity-webgl.dev/docs/api/send-message

**Reply by harmanpreet7916** [reply]
Thank you @jeffreylanters for replying this fast. This works, also apologies for the not seeing documentation.

**Reply by jeffreylanters** [reply]
No need to apologize! Good luck with your project!
